projects/control-service/projects/job-builder/Dockerfile

```diff
@@ -4,19 +4,22 @@ FROM gcr.io/kaniko-project/executor
 
 FROM alpine
 
-COPY --from=0 /kaniko /kaniko
-
+COPY --chown=1000:2000 --from=0 /kaniko /kaniko
```

Comment:
Make user and group configurable and set the default values - respectively user=1000 and group=2000.